src/types/loans.ts

```diff
@@ -15,6 +15,22 @@ interface BorrowPosition extends LoanPosition {
     accumulatedDebt: MonetaryAmount<CurrencyExt>;
 }
 
+type LoanAction = "lend" | "withdraw" | "borrow" | "repay";
+interface LoanCollateralInfo {
```

Comment:
This interface contains information about a lot of aspects of the lending protocol, and `LoanCollateralInfo` is misleasing. I'd rename more generically to just `LoansStatus` or similar.

Reply:
Naming this one is not easy, `LoansStatus` is already used on UI to denote risk of getting liquidated, so I would use `LoansAccountInfo`, what do you think about that one? Open to other suggestions!

Reply:
`LendingStats`?

Reply:
sounds good :+1: